packages/datadog-plugin-mocha/src/index.js

```diff
@@ -47,7 +47,7 @@ class MochaPlugin extends Plugin {
   constructor (...args) {
     super(...args)
 
-    this._testSuites = new WeakMap()
+    this._testSuites = new Map()
```

Comment:
Why was this a WeakMap? If it shouldn't be holding memory then this is a significant change.

Reply:
we used to use the `suite` instance but now the key is a test file (string)